src/anm/player.js

```diff
@@ -377,6 +377,8 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
             state.happens = C.RES_LOADING;
             player.fire(C.S_CHANGE_STATE, C.RES_LOADING);
             player.fire(C.S_RES_LOAD, remotes);
+            // actually start loading remote resources
+            anim._loadRemoteResources(player);
```

Comment:
I look in this case now and this way we get a separate new subscribers to an url, because RM logic is designed like: 

* Someone's subscribes to a bunch resources
* All resources trigger to start loading
* If they some of them are in cache, just give them directly to subscribers. And if they are in cache, no `check()` method should be called, see:

https://github.com/Animatron/player/blob/develop/src/anm/resource_manager.js#L98
https://github.com/Animatron/player/blob/develop/src/anm/resource_manager.js#L139

But in the new case it goes like:
* Start loading all resources (loading is started synchronously, of course, then for most of the cases goes asynchronous)
* Also, in loaders, a separate new subscriber is created for every URL, since no-one subscribed to it for now, and `me._waiting[url]` is `null` [here](https://github.com/Animatron/player/blob/develop/src/anm/resource_manager.js#L102), so it jumps to [this]
(https://github.com/Animatron/player/blob/develop/src/anm/resource_manager.js#L121) branch of code.
* Subscription is performed (after resources started loading).

I think if `check()` was a bottleneck, then it should be optimized, but subscribing after triggering the load process looks illogical for me.

I feel I'd have to revert it back for now, since it breaks the RM logic for me.